draft-huitema-quic-in-space.md

```diff
@@ -78,6 +78,12 @@ network management considerations are out of scope of this document.
 
 {::boilerplate bcp14-tagged}
 
+# Path Changes
+
+In deep space in a direct communication scenario such as an Earth ground station to directly an asset on Mars, the RTT is large but pretty stable. However, when an orbiter acting as a communication relay is used, the RTT has a very different pattern. As a simple example scenario, Earth communicates to orbiter which communicates to a Mars asset and vice-versa, Mars is at its nearest distance to Earth (4 minutes delay) and the orbiter takes 2 hours to make a circular orbit, which means 1 hour reachable from Earth. In this scenario, the Earth ground station will see a stable RTT (~8 minutes) to the orbiter when the orbiter is directly reachable from Earth and the asset is reachable from the orbiter. When the orbiter is no more reachable as it is going on the far side, the ground station will accumulate packets over time, and then when the orbiter becomes reachable again, will send those buffered packets. The first buffered packets just after the orbiter is unreachable will have 1 hour + 8 minutes RTT. The last buffered packets just before the orbiter becomes reachable will have a 8 minutes RTT. In between, it will be essentially linear. In this simple scenario, the RTT shows a sawtooth pattern with some stable value period. Obviously, depending on the location of the asset, the type of orbit, the number of orbiters and assets, etc, that pattern will be more complex and different. However, it will still show an important jump, like from 8 minutes to 1 hour, in the RTT.
+
```

Comment:
I will break that very long paragraph into multiple lines.

I think there is an issue of facts. If we look at the Moon, it is tidal-locked to the Earth, so what you say is correct: on the hidden face of the Moon, communication is always impossible; on the visible space, it should be always possible if we have enough stations on earth to balance the rotation of Earth. But for Mars, we have a different issue: Mars rotates around itself. Without orbiters, the station on Mars is only able to communicate with Earth for half of the Martian day.

Now, with orbiters, we do have a different pattern. If the orbiters acts as a store and forward relay, it will store packets sent from the "hidden face" during half the orbit, and forwards them when it emerges on the visible side.